modules/postgresql/src/main/java/org/testcontainers/containers/PostgreSQLContainer.java

```diff
@@ -35,7 +35,7 @@
     private static final String QUERY_PARAM_SEPARATOR = "&";
 
     /**
-     * @deprecated use {@link PostgreSQLContainer(DockerImageName)} instead
+     * @deprecated use {@link PostgreSQLContainer(String)} instead
```

Comment:
Actually I see the problem, but the provided change also suffers from it.

```suggestion
     * @deprecated use {@link PostgreSQLContainer#PostgreSQLContainer(DockerImageName)} or 
     * {@link PostgreSQLContainer#PostgreSQLContainer(String)} instead
```

I think it needs to be like this, adding `#PostgreSQLContainer` to clarify that this is a constructor reference. Also, we may as well point to both the available non-deprecated constructors.

Reply:
Right; I've had it fixed (ammended this change to the commit).

Reply:
Thanks! 😄